### DISCUSSION TITLE
beaker browser archived

### DISCUSSION THREAD

**Discussion by fcjz** [TiddlyWiki/TiddlyWiki5#7218] [General]
The repository of Beaker Browser has been archived, the project has lost its maintenance, and the official website can no longer be opened. Even if the previous application can still be used, should it be removed from the way of use?

**Comment by pmario** [comment]
I think we should remove it

**Comment by Jermolene** [comment]
Thanks @fcjz I've removed the docs references in 485bf19c3cbfb8c22cd34f26a6642598f2d99630 and the savers themselves in 5d650e87dd6ba66829c7e29b39b1eb3069a78ebf
